src/trusted/service_runtime/nacl_app_thread.c

```diff
@@ -653,6 +694,17 @@ int NaClAppThreadSpawn(struct NaClAppThread     *natp_parent,
     NaClTlsSetTlsValue2(natp_child, user_tls2);
   }
 
+
+  if (tl_type != THREAD_LAUNCH_THREAD) {
+    natp_child->is_cage_parent = true;
```

Comment:
I'd prefer not to use the word parent in the context of threads, just for clarity. Main thread I think is clearer?